cost/kubecost/cluster/kubecost_cluster_rightsizing_recommendations.pt

```diff
@@ -291,12 +329,7 @@ end
 
 policy "pol_kubecost_recommendations" do
   validate_each $ds_recommendations do
-    summary_template "{{ with index data 0 }}{{ .policy_name }}{{ end }}"
-    detail_template <<-'EOS'
```

Comment:
Why are we removing this? What does this have to do with updating API endpoints?

Reply:
I reverted this one also